examples/window_option_as_alt.rs

```diff
@@ -0,0 +1,69 @@
+#![allow(clippy::single_match)]
+
+#[cfg(target_os = "macos")]
+use simple_logger::SimpleLogger;
+
+#[cfg(target_os = "macos")]
+use winit::platform::macos::WindowExtMacOS;
+
+#[cfg(target_os = "macos")]
+use winit::{
+    event::{ElementState, KeyboardInput, VirtualKeyCode},
+    event::{Event, WindowEvent},
+    event_loop::EventLoop,
+    window::WindowBuilder,
+};
+
+#[cfg(target_os = "macos")]
+fn main() {
+    SimpleLogger::new().init().unwrap();
+    let event_loop = EventLoop::new();
+
+    let window = WindowBuilder::new()
+        .with_title("A fantastic window!")
+        .with_inner_size(winit::dpi::LogicalSize::new(128.0, 128.0))
+        .build(&event_loop)
+        .unwrap();
+
+    let mut option_as_alt = false;
+
+    event_loop.run(move |event, _, control_flow| {
+        control_flow.set_wait();
+        // println!("{:?}", event);
+
+        match event {
+            Event::WindowEvent {
+                event: WindowEvent::CloseRequested,
+                window_id,
+            } if window_id == window.id() => control_flow.set_exit(),
+            Event::WindowEvent { event, .. } => match event {
+                WindowEvent::KeyboardInput {
```

Comment:
Nit: React to mouse input instead, since this option modifies how keyboard input works

Reply:
Updated this whole example to be more useful.